hibike/devices/BatteryBuzzer/battery_buzzer.h

```diff
@@ -1,56 +1,32 @@
-#ifndef EX_DEVICE_H
-#define EX_DEVICE_H
+#ifndef BATTERY_BUZZER_H
+#define BATTERY_BUZZER_H
 
 #include "hibike_device.h"
-#include "pitches.h"
+#include "eeprom.h" //implement
+#include "safety.h" //implement
+#include "voltage_sense.h"
+#include "disp_8.h"
 
-#define READ_ENABLE_PIN 9
-#define BUZZER_PIN 10
-#define BUZZER_LED_PIN 11
-#define BUZZER_FREQ NOTE_A4
-#define BUZZER_DISCONNECT_FREQ NOTE_A3
-#define BUZZER_BROKEN_FREQ NOTE_DS2
 
-#define NUM_CELLS 3
-#define BATT_0 A0
-#define BATT_1 A1
-#define BATT_2 A2
-#define BATT_0_SCALE 2.0  // From voltage divider
-#define BATT_1_SCALE 4.0
-#define BATT_2_SCALE 6.1
-#define BATT_INPUT {A0, A1, A2}
-#define BATT_CHANNELS {7, 6, 5}  // The ADC channels for A0, A1, A2 on atmega32u4
-#define NUM_SAMPLES 52  // Average this many readings per cell voltage, should not be greater than 63
-// units analogRead() uses
-#define UNITS_PER_VOLT (1023.0/5.0)  // From ADC
-#define PRESCALE (1/(NUM_SAMPLES*UNITS_PER_VOLT))
-#define DEFAULT_CALIBRATION {PRESCALE*BATT_0_SCALE, PRESCALE*BATT_1_SCALE, PRESCALE*BATT_2_SCALE}
-#define BLINK_AFTER_MEASURE 10  // At 2Hz
+extern float v_cell1;
```

Comment:
There's a lot of externs here.  SInce this is now a cpp file, we can move away from keeping all globals in the main file.